woocommerce/google-listings-and-ads
# Add gender attribute

### Changes proposed in this Pull Request:

As part of #16

This PR adds a new product attribute called Gender for all products and product variations. This attribute has 3 valid values presented to the user as a select box to choose from: `male`, `female`, and `unisex`. It also has a `Default` value similar to all other attributes which leaves the value to be filled by a default value set in the settings (not yet implemented).

### Detailed test instructions:

1. Edit a product or create a new one
1. Choose a gender for the product via the edit product page (under the Google Listings and Ads section of product data)
5. Publish or update the product and wait for it to sync (or sync manually via the Connection Test page)
6. Check Google Merchant Center and make sure the gender attribute is set correctly for the product
7. Repeat the above steps for a variable product and make sure the brand is set correctly for all variations (for variations it defaults to the value set in the parent product)
